Wallet.md

```diff
@@ -0,0 +1,52 @@
+# BTCPay Server Wallet
+This document explains the features of the internal BTCPay wallet. For frequently asked questions - check the [Wallets FAQ](FAQ-Wallet.md)
+
+Note: A [private key](https://en.bitcoin.it/wiki/Private_key) (xprv)is **never** required for BTCPay to work. The software needs a public key (xpub) which is a watch-only wallet token. The xpubkey allows BTCPay to generate a new address each time a new invoice is created. It enables users to observe the wallet balance and transactions without the need for a private key.
+
+If you're using a [third-party host](ThirdPartyHosting.md) and are asked to provide a private key, it's certainly a scam. Never share your private key with anyone. It's called private for a reason. 
+
+BTCPay has a**internal**, full node reliant wallet that can be accessed by clicking "Wallets" from the top menu. The reliance on the users own full node makes the wallet more private and secure compared to any [SPV](https://en.bitcoin.it/wiki/Thin_Client_Security#Simplified_Payment_Verification_.28SPV.29) wallet.
+
+The internal wallet does not generate a private key. Users have to create their own private key and enter xpubkey derived from xpriv into Store Settings > Derivation scheme to activate the internal wallet. 
+
+The public key can be easily added from external wallet like Electrum by pasting it into derivation scheme field or automatically with a hardware wallet like Ledger Nano S. For more information on how to add a derivation scheme to BTCPay, see the [Getting Started Guide](GettingStarted.md)
```

Comment:
```suggestion
The xpub key can be easily added from external wallet like Electrum by pasting it into derivation scheme field or automatically with a hardware wallet like Ledger Nano S. For more information on how to add a derivation scheme to BTCPay, see the [Getting Started Guide](GettingStarted.md)
```

Reply:
xpub matches your sentence above "enter the xpubkey... in store settings" and what is shown on the derivation scheme page for BTCPay but then in Electrum it's 'Master Public Key' so I'm not sure, up to you.